### ISSUE TITLE
Add task for releasing a staging repository

### ISSUE THREAD

**Issue by marcphilipp** [gradle-nexus/publish-plugin#6]

**Comment by ihostage** [comment]
Close/Release repository tasks must have ordering settings.
For details Codearte/gradle-nexus-staging-plugin#157

**Comment by szpak** [comment]
@ihostage It's already [implemented[(https://github.com/gradle-nexus/publish-plugin/blob/master/src/main/kotlin/io/github/gradlenexus/publishplugin/NexusPublishPlugin.kt#L148-L156) as part of [this PR](https://github.com/gradle-nexus/publish-plugin/pull/9).

I will update the tasks based on my recent development effort with releasing.

**Comment by ihostage** [comment]
Thank a lot, @szpak 👍

**Comment by szpak** [comment]
Implemented in #18.

```
close${repositoryName()}StagingRepository
release${repositoryName()}StagingRepository
```
